package.json

```diff
@@ -54,24 +24,58 @@
     "object",
     "array",
     "check",
-    "test"
+    "test",
+    "typescript"
   ],
-  "nyc": {
-    "lines": 100,
-    "statements": 100,
-    "functions": 100,
-    "branches": 100,
-    "include": [
-      "src/*.js"
-    ],
-    "exclude": [
-      "src/*.test.js"
-    ],
-    "reporter": [
-      "lcov",
-      "text-summary"
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/moroshko/shallow-equal.git"
+  },
+  "license": "MIT",
+  "author": "Philip Su <philip@sparkanvil.com>",
```

Comment:
@moroshko : only making this change because you're busy. Happy to revert to yours if you'd still prefer to be listed!

Reply:
No worries, I don't mind!